### PR TITLE
replace generate_grammar.py by generate-grammar.vsh

### PR DESCRIPTION
**Pull request by gechandesu**
This PR:
- Replaces Python code with .vsh script
- Removes generate_grammar.py mention from docs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ vsql-server
 grammar.bnf
```
Comment (elliotchance) [inline .gitignore:7]: Actually, I forgot to remove these as well:

```
grammar.bnf
vsql/grammar.v
```

Also, `scripts/generate-grammar` does not need to be added?

File: scripts/generate-grammar.vsh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env -S v
```
Comment (elliotchance) [inline scripts/generate-grammar.vsh:1]: You'll also need to update the Makefile that points to this. Make sure `make grammar` is working

**Review by elliotchance — CHANGES_REQUESTED** [review]

File: scripts/generate-grammar.vsh (hunk 2 above)
Comment (gechandesu) [reply scripts/generate-grammar.vsh:1]: Done

**Review by elliotchance — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (gechandesu) [reply .gitignore]: `scripts/generate-grammar` should be ignored, this file is automatically built on .vsh run, no reason to keep it in git so binary is dynamically-linked and not portable.

**Pull request merged**